specification/gp-connect-user-permissions.yaml

```diff
@@ -10,9 +10,21 @@ info:
     email: api.management@nhs.net
   description: >
     ## Overview
-    Add a fully fledged description here with markdown syntax.
-    For help completing the content of your API spec, see our guidance on the minimum standard (bronze) details
-    to include [here](https://nhsd-confluence.digital.nhs.uk/display/APM/Writing+a+bronze+API+cover+page). You should also base your content on our exemplar API - [PDS FHIR](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir).
+    
+    Use this API to access an NHS patient's permissions for healthcare information and services. Patient facing applications can get a patient's permissions from GP Systems with this API.
+    
+    If a patient has an NHS Login ID, you can:
```

Comment:
Do this need to also state that they must be a member of a GP practice or is that just a given?

Reply:
From looking at other specs I think they prefer a very simple overview. Perhaps this should instead just say "You can:" and the detail of the patient requirements can go elsewhere?

Reply:
Sure, sounds good.